### PR TITLE
Use macaw sidebar

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it moves sidebar declaration to macaw.

**PR intended to be tested with API branch:** master

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3396 Extract sidebar to Macaw UI](https://app.clickup.com/t/2549495/SALEOR-3396)

**Comment by patrys** [comment]
Task linked: [SALEOR-3287 Use Macaw-UI in both Saleor Cloud and Saleor Dashboard](https://app.clickup.com/t/2549495/SALEOR-3287)

**Review by orzechdev — APPROVED** [review]

File: src/components/AppLayout/utils.ts [hunk 1]
```diff
@@ -1,8 +1,7 @@
+import { IMenuItem } from "@saleor/macaw-ui";
```
Comment (jwm0) [inline src/components/AppLayout/utils.ts:1]: We should be getting rid off the `I` prefixed interfaces whenever we find the chance

Comment (dominik-zeglen) [reply src/components/AppLayout/utils.ts:1]: Will do :+1:

Comment (dominik-zeglen) [reply src/components/AppLayout/utils.ts:1]: Since it's already merged I created ticket to fix it later
https://app.clickup.com/t/mh6qyx

**Review by jwm0 — APPROVED** [review]

**Pull request merged**